terraform/modules/tf_cloudwatch/iam.tf

```diff
@@ -1,9 +1,14 @@
 # IAM Role: Allows CloudWatch Logs to put data into
 # this cluster's default Kinesis stream
 resource "aws_iam_role" "cloudwatch_subscription_role" {
-  name = "stream_alert_${var.cluster}_cw_sub_role_${var.region}"
-
+  name               = "${var.prefix}_${var.cluster}_cloudwatch_subscription_role"
+  path               = "/streamalert/${var.region}/"
```

Comment:
Aha, this is for cloudwatch, @ryandeivert

Reply:
this exists because the way this module is structured, it created a subscription in each region. instead of 'baking' that into the resource name, it felt better to use it in the path prefix instead